setup.cfg

```diff
@@ -80,7 +80,7 @@ all =
     pytz
     jplephem
     mpmath
-    asdf>=2.6
+    asdf@git+https://github.com/asdf-format/asdf.git
```

Comment:
Tests are only passing with this pin, so that means we either have to:

* Pin `pytest<7`

*OR*

* Wait for `asdf` to release and then bump minversion of `asdf` to that release.

This pin broke RTD but that is okay because we will not merge with this pin anyway.